build/set-env.ps1

```diff
@@ -39,10 +39,18 @@ If (-not (Test-Path -Path $Env:WHEEL_OUTDIR)) { [IO.Directory]::CreateDirectory(
 If ($Env:DOCS_OUTDIR -eq $null) { $Env:DOCS_OUTDIR =  (Join-Path $Env:DROPS_DIR "docs") }
 If (-not (Test-Path -Path $Env:DOCS_OUTDIR)) { [IO.Directory]::CreateDirectory($Env:DOCS_OUTDIR) }
 
+$env:BUILD_PLATFORM = "win-x64"
```

Comment:
Fair question... it would default to using the folder name `win-x64` and would likely break. But it would also break in a lot of other places, such as our CMakefile.txt build infrastructure that similary assumes one of those platforms. My understanding is that anything other than those three is not part of our support statement at the moment, so lots of work would have to be done throughout to make our repo build compatible outside of those three.

Reply:
Makes sense, as I said I'm surely being too paranoid here. Just gotten wary of fallthrough control flow...